### PR TITLE
Update project README / metadata for release prep

### LINKED ISSUE #45: There is a new release of the Conjur Ansible role
AC: 
- [ ] Update the README to be clear that we recommend users use this project for Ansible 2.8, and refer to the collection for Ansible 2.9+
- [ ] Review changelog / state of project pre-release
- [ ] Add NOTICES if necessary

### PR DESCRIPTION
**Pull request by izgeri**
### What does this PR do?
The first commit in this PR does some generic cleanup:
- Add version file
- Add release instructions to CONTRIBUTING
- Add script to publish to Galaxy
- Add publish stage to pipeline
- Update README to use valid links and standard configuration var defs; to
  recommend using the collection for Ansible 2.9+
- Update the Summon and Summon-Conjur defaults to the latest versions
- Update the metadata to have general maintainer authors and use 2.8 as the
  minimum version (all older versions are already EOL)

The second commit increments the version to prepare for release.

### What ticket does this PR close?
Resolves #45 

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,75 @@
...
+   to. When connecting to an HA DAP master cluster, this would be the URL of the
+   master load balancer.
+* `conjur_account` `*`: The account name for the Conjur instance you are connecting to.
+* `conjur_host_factory_token` `*`: [Host Factory](https://docs.conjur.org/Latest/en/Content/Operations/Services/host_factory.html)
+  token for layer enrollment. This should be specified in the environment on the
+  Ansible controlling host.
+* `conjur_host_name` `*`: Name of the host identity for the host factory to create.
+* `conjur_ssl_certificate`: The PEM-encoded x509 CA certificate chain for the DAP
+  instance you are connecting to. This value may be obtained by running the command:
+  ```
```
Comment (diverdane) [inline README.md:62]: I'm seeing CodeClimate warnings re. "Fenced code blocks should be surrounded by blank lines" for this line (62) and (64). There are also some "Line length" warnings, some of which are unavoidable due to long URLs.

File: README.md [hunk 2]
```diff
@@ -65,11 +85,13 @@ Configure a remote node with a Conjur identity and Summon:
 ```
 
 This example:
-* Registers the host with Conjur, adding it into the layer specific to the provided host factory token.
-* Installs Summon with the Summon Conjur provider for secret retrieval from Conjur.
+* Registers the host with Conjur, adding it into the layer specific to the provided
```
Comment (diverdane) [inline README.md:89]: CC says "Lists should be surrounded by blank lines".

**Review by diverdane — APPROVED** [review]
This LGTM!
Though I am seeing some CodeClimate warnings about line length, and fenced code blocks and lists not being surrounded by blank lines in the `README.md`, if we care about these things.

File: README.md (hunk 1 above)
Comment (izgeri) [reply README.md:62]: I'm not sure the formatting works properly if I surround it with blank lines, so I'll leave it as-is for now

**Pull request merged**